src/AuthenticationMiddleware.php

```diff
@@ -18,10 +18,13 @@
 class AuthenticationMiddleware implements MiddlewareInterface
 {
     /**
-     * @var AuthenticationInterface
+     * @var AuthenticationInterface|\Zend\Expressive\Authentication\AuthenticationInterface
```

Comment:
Expanding the type on a `protected` property is a BC break: is this change needed?

Reply:
I agree with you.
https://github.com/mezzio/mezzio-authentication/blob/9d3e47e4e9e5081296be2a7bc3c5dea2fb01a58c/src/AuthenticationMiddlewareFactory.php#L19-L23

But `$authentication` can be `AuthenticationInterface` or `\Zend\Expressive\Authentication\AuthenticationInterface`. 🤔

Reply:
The `zend`-prefixed one is legacy and handled by class aliasing, so it shouldn't be required anymore

Reply:
Can I revert this changes?

Reply:
Yes

Reply:
> Yes

Done.